whats-new/whatsnew_en.md

```diff
@@ -1,22 +1,7 @@
-
-## Ease full raw data drill down
-The drill down from the high-level overview dashboard to the details of a report to the raw data aggregated helps to investigate and analyze findings. This navigation is already possible today but difficult to follow. This release now streamlines the navigation. Each report on a dashboard can be clicked anywhere on the report tile to drill down to the report, which now contains the report plus the raw data side by side. 
-
-## Send Optimize notifications to the notification tracker
-Optimize is now sending important notifications to the Camunda 8 notifications tracker. This allows us to support users utilizing Camunda 8 for process orchestration with hints about the benefits of using Optimize.
-
-## Copy Instant Process Dashboard
-When viewing an Instant Process Dashboard, you can now directly create a copy that can then be edited and shared like any other dashboard. This dashboard is now also available as a selectable template during dashboard creation.
-
-## Updated Report Templates
-Report Templates help the user to visualize process execution data in a shareable report quickly. The new Report Templates provide an overview of the different report visualizations in Optimize.
-
-## Descriptions for reports and dashboards (All distributions)
-Describing how to “read” visualized data and the context of why exactly this data is displayed greatly improves the readability of reports and dashboards. With 3.10, we introduced Text Reports that can be added as tiles, similar to other reports, and allow to place descriptions directly beside reports. This release added description fields directly related to reports and dashboards. 
-
 ## Carbon Design migration (All distributions)
+
 We are in the process of migrating Camunda Optimize UI to use the Carbon Design System. Our primary focus is to bring the Optimize UI in line with other Camunda products, providing a consistent and seamless experience across the entire ecosystem. With this migration, we will also adhere to accessibility best practices, ensuring that our application is usable by a broader range of users.
 
 The migration to the Carbon Design System for Camunda Optimize is an ongoing effort. We have successfully migrated modals, tables, and popovers, ensuring these components seamlessly integrate into the new design. While the functionality you love remains intact, you can expect a refreshing new UI that aligns with the Carbon Design System.
+
+The updated Task Analysis now includes a revamped outliers table, offering enhanced insights into your processes. This table provides detailed information about outlier process instances, including the number of instances and related variables. For even more in-depth details, you can explore the modal, which showcases outliers on the duration chart and in the table.
```

Comment:
I guess this section needs a title.

Reply:
Yes, I posted this on the task analysis epic channel and he approved it